railties/lib/rails/generators/rails/plugin/templates/bin/webpack-dev-server.tt

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
```

Comment:
I would like to avoid include these files in the Rails repository. If the file generated by Webpacker is changed, it may generate a file different from the file expected by Webpacker.

I think that it is appropriate to add an installation task for Rails engines on Webpacker.

Reply:
I agree. I had the same concern but I had to add these files here because running any rake after adding the plugin throws gem validation error. I tried to check how to bypass that validation but could not find anything.

```
blorgh/blorgh.gemspec is not valid. Please fix this gemspec. (Gem::InvalidSpecificationException)
The validation error was '"FIXME" or "TODO" is not a description'
```

Reply:
Is it not good just to show message like "If you want to use webpacker, please run the installation task of webpacker" after created plugin? 
Unlike an application, `bundle install` is also not executed, so I think it is not unnatural that installation tasks are not automatically executed.

Reply:
If I understand it correctly, we will add the webpacker configuration in the engine files and helpers as suggested by the guide but in order to get the binstub and webpacker env config we will prompt the user to fix the gemspec and run the rake task. Is that correct? 

I did not think of this solution as I was making webpacker as default for engines but I can work on it.

Reply:
IMHO all files should be generated/edited by Webpacker's task, not Rails. I think that Rails should not generate files that dependent on Webpacker.

Reply:
Yes, we should generate those file using webpacker.